assets/templates/plan-calculator.html

```diff
@@ -183,18 +183,19 @@ <h4>Suscripción</h4>
                         </article>
                       </div>
                       <div class="col-md-6 col-sm-12">
-                        <div class="dp-wrap-price">
-                          <span class="dp-old-price"
-                            ><span class="dp-money">US$</span
-                            ><span class="dp-old-amount">15</span></span
+                        <!--uno -->
+                        <div class="dp-price--wrapper">
+                          <span class="dp-price-old"
+                            ><span class="dp-price-old-money">US$</span
+                            ><span class="dp-price-old-amount">15</span></span
                           >
-                          <h2 class="dp-amount-large">
-                            <span class="dp-money">US$</span
-                            ><span class="dp-amount">11,25</span>
+                          <h2 class="dp-price-large--amount">
```

Comment:
```suggestion
                          <h2 class="dp-price-large-amount">
```
The class `dp-price-large--amount` does not exist, either change it in the name or in the html